### DISCUSSION TITLE
Rendering short schedules (15 min)

### DISCUSSION THREAD

**Discussion by vozi314** [nhn/tui.calendar#1076] [Q&A] [answered]
Hello, 
i would like to ask about following:
![schedules](https://user-images.githubusercontent.com/7164797/162690894-96d0b548-7321-4cc4-b22e-3f083aceeb6b.png)

Why are schedules rendered with small negative vertical offset, witch cases that schedules are shown in 3 columns instead of one as I would expect. Schedules are 12:00 - 12:15, 12:15 - 12:30, 12:30 - 12:45. I was trying to figure it out by theme config, but with no success. Did I missed something? 

Great thanks in advance

**Comment by adhrinae** [comment, marked as answer]
Please refer to #563.
